server/routes/signOut.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+
+module.exports = function createRouter() {
+  const router = express.Router();
+
+  router.get('/', (req, res) => {
+    req.logout();
```

Comment:
Do we need to clear out anything on the client side too?

Reply:
Good point :)